AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -813,11 +813,7 @@ - (BOOL)displaysAsynchronously
 - (BOOL)_displaysAsynchronously
 {
   ASDisplayNodeAssertThreadAffinity(self);
-  if (self.isSynchronous) {
-    return NO;
-  } else {
-    return _flags.displaysAsynchronously;
-  }
+  return self.isSynchronous = NO && _flags.displaysAsynchronously;
```

Comment:
`self.isSynchronous = NO` looks like a minor mistake. We want to compare the value instead of assigning?